index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <nav id="navcontainer">
+    <div>
+      <h3 id="menulogo">John Doe</h3>
```

Comment:
- [ ] Instead of using an h3 it's one of the requirements to use an `<a></a>` tag, kindly fix that.
- [ ] [_OPTIONAL_] Since you're building this website to serve as a portfolio you can put your name instead of John Doe.